app/models/user.rb

```diff
@@ -25,7 +25,7 @@ class User < ActiveRecord::Base
   has_many :collection_roles, through: :identity_group
   has_many :user_seen_subjects
 
-  has_many :subject_queues, dependent: :destroy
```

Comment:
Did you remove this because we don't actually delete users? I assume if we ever did then we'd clean up their queues.